src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -132,6 +132,12 @@ export default {
 		}
 	},
 
+	computed: {
+		hasListSlotItems() {
+			return this.$scopedSlots.list?.()?.length > 0
+		},
+	},
```

Comment:
`$scopedSlots` is not reactive and render function should only be used in rendering (another render function or template). Thought this `computed` is only used in template, it's safer to directly check for a list in the template with `v-if="$scopedSlots.list"`.

Is there a case there we really need to check the content of the slot, not only the slot presence?

Reply:
> Is there a case there we really need to check the content of the slot, not only the slot presence?

Nope, fixed!